trio/_core/tests/test_local.py

```diff
@@ -166,10 +166,93 @@ def test_local_outside_run():
         nursery.start_soon(spawner, nursery)
 
 
-async def test_local_defaults():
+async def test_local_defaults(recwarn):
     for cls in _core.TaskLocal, _core.RunLocal:
         local = cls(default1=123, default2="abc")
         assert local.default1 == 123
         assert local.default2 == "abc"
         del local.default1
         assert not hasattr(local, "default1")
+
+
+# scary runvar tests
+def test_runvar_sanity():
```

Comment:
Style nit: can you rename this `test_runvar_basics` or `test_runvar_smoketest`? I try to avoid using "sanity" in the jokey "not horribly broken" sense these days b/c the jokey sense seems to come from actually pretty gross stereotypes about mental health. (And same comment on the other uses of "sanity".)